monGARS/core/caching/tiered_cache.py

```
@@ -1,56 +1,100 @@
-async def get_data_from_disk_cache(param1: str, param2: int):
-    logger.info(f"Loading data from source with params: {param1}, {param2}")
-    await asyncio.sleep(5)
-    return {"param1": param1, "param2": param2, "source": "disk"}
\ No newline at end of file
+import logging
+import pickle
+from pathlib import Path
+from typing import Any
+
+from aiocache import Cache, caches
+from aiocache.serializers import PickleSerializer
```

Ruff (0.12.2)

`aiocache.serializers.PickleSerializer` imported but unused

Remove unused import: `aiocache.serializers.PickleSerializer`

(F401)